cohort_3/curriculum/workshops/portfolio/README.md

```diff
@@ -0,0 +1,271 @@
+# Portfolio
+
+## Cloud9
+
+- Go to https://c9.io, click the `Sign Up` button, and create an account
+- Click `Go to your dashboard`
+- Creating a workspace
+    - Click `Create a new workspace`
+    - Give it the name `hack-camp`
+    - Make sure the `Custom` type is selected
+    - Click `Create workspace`
+- Configuring your workspace
+    - Select the `Minimal Editor` preset.
+    - Change `Soft Tabs` to 2
+    - Turn on `Enable Auto-Save`
+    - At the very top of the window, click the little triangle on the bar to
+      show the Cloud9 editor options.
+
+      ![](img/c9_configuring_workspace_top_bar.png)
+
+    - Click `View` and then `Console` to open up the terminal
```

Comment:
It hides the IDE features on the right sidebar - though there may be a better way to configure that.